openpype/hosts/blender/api/ops.py

```diff
@@ -1048,6 +1048,9 @@ def _build_first_workfile(self, clear_scene: bool, save_as: bool):
             # clear all libraries
             for library in list(bpy.data.libraries):
```

Comment:
We could change that the correct way by the same move

```suggestion
            for library in bpy.data.libraries:
```